client/shipping-label/views/index.js

```diff
@@ -49,15 +52,49 @@ class ShippingLabelRootView extends Component {
 		this.setState( { showTooltips } );
 	}
 
+	renderPaymentInfo() {
+		const paymentMethod = this.props.shippingLabel.paymentMethod;
+
+		if ( paymentMethod ) {
+			return (
+				<Notice isCompact={ true } showDismiss={ false } className="wcc-metabox-label-payment inline">
+					<p dangerouslySetInnerHTML={ { __html: sprintf(
```

Comment:
String translations need to contain tokens for string substitutions so they can be translated for RTL languages. If we append the card digits, we break that.